scan.go

```diff
@@ -671,6 +694,18 @@ func (zp *ZoneParser) Next() (RR, bool) {
 	return nil, false
 }
 
+// canParseAsRR returns true if the record type can be parsed as a
+// concrete RR. It blacklists certain record types that must be parsed
+// according to RFC 3597 because they lack a presentation format.
+func canParseAsRR(rrtype uint16) bool {
```

Comment:
I would invert this and rename it to something like `mustParseAsGeneric`, using vocabulary from [RFC 3597](https://tools.ietf.org/html/rfc3597#section-5).

Reply:
If I touch this again in a future PR, I can certainly consider doing that.